Transparency in the Interoperability of the National Electronic Toll(ETC) network (#203)
hyperledger-labs/hyperledger-labs.github.io: labs/electronic-toll-collection-system.md

---
layout: default
title: Transparency in the Interoperability of the National Electronic Toll(ETC) network
[//]: # (parent: Labs)
---
# Lab Name
Transparency in the Interoperability of the National Electronic Toll(ETC) network

# Short Description
Hybrid Blockchain Network that seeks the route of Transparency, Traceability and Correct collection Payment between operators at Electronic Toll Collection(ETC)

# Scope of Lab

> Derived from the current technical situation, the development of an interface capable of being able to insert each of the crossings made at the toll booths through electronic tolls in a private blockchain network is proposed, where in the same way it can send data of control figures to a network. public; Once the previous steps have been carried out, you can immediately filter and determine which operator the toll belongs to, complying with the interoperability process, leaving SFTP servers out, and adapting a queuing system that can comply with fault tolerance.

Reviewer: ```suggestion
Derived from the current technical situation, the development of an interface capable of being able to insert each of the crossings made at the toll booths through electronic tolls in a private blockchain network is proposed, where in the same way it can send data of control figures to a network. Once the previous steps have been carried out, you can immediately filter and determine which operator the toll belongs to, complying with the interoperability process, leaving SFTP servers out, and adapting a queuing system that can comply with fault tolerance.
```

> In order to comply with transparency in the transactions carried out and if it is necessary to request information directly from the private network to reconcile information that the regulatory entity may question, where the private blockchain will be its main source of information, NEAR blockchain will be used. public for the issuance of delivery document receipt of requested information, generating document as NFT, and can be used as unique proof, using IPFS as a file storage system, where in the same way the integrity of the generated document can be counted on.

Reviewer: This paragraph does not read properly. The second sentence starts with "public for the issuance of delivery document..." It seems there is something missing in the description.

---
layout: default
title: Transparency in the Interoperability of the National Electronic Toll(ETC) network

> [//]: # (parent: Labs)

Reviewer: ```suggestion
parent: Labs
```

In order to comply with transparency in the transactions carried out and if it is necessary to request information directly from the private network to reconcile information that the regulatory entity may question, where the HyperLedger Fabric blockchain will be its main source of information, NEAR blockchain will be used to public for the issuance of delivery official document receipt and receive of requested information, generating document as NFT, and can be used as unique proof, using IPFS as a file storage system, where in the same way the integrity of the generated document can be protected.

The government information is very sensible, and the message ETC is a root information to create invoice where federal institutions receive incomes and after they need to make a payments to operators companies by services of leasing from IoT devices, and adminstration payments too.

As we can see, ETC message is very important to make concilian information to differents participants with some confusions about charges made or authorities whom receive incomes from differents operators companies.

> Hyperledger fabric has a important role to keep and save information with principal objective message ETC, acomplish with this, inmutability, integrity and to government and some private companies, they can have control about their own information.

Reviewer: ```suggestion
Hyperledger Fabric has an important role to keep and save information with principal objective message ETC, accomplish with this, immutability, integrity and to government and some private companies, they can have control about their own information.
```

Continuing with the flow of information transparency, said funds transferred through the NEAR token will continue to be transparent for the client, since they will be in a public network, said protocol was selected, since the cost of transactions is lower. to others such as ETHEREUM, in addition to the use of the Proof Of Stake (PoS) protocol, making transactions per second (TPS) more agile and faster.

The benefits of the solution are several, where the most important are:

1.- Transparency and Integrity in the information collected from the costs of the operators,

2.- If there is a loss of communication, it is avoided having to make unnecessary queries to the Ledger or DB engine, since the management of queues applies fault tolerance concepts.

3.- It is guaranteed that the interoperability tags are delivered to the operator that belongs to it, removing the SFTP servers.

4.- Information may be reconciled by operators, if the figures do not coincide with what is requested by the authority.

5.- Information on control figures can be uploaded every month end.

> ![INTER-OPERABILIDAD-WorkFlow_ ETC_ NEAR - GATEWAY (4)](https://user-images.githubusercontent.com/11085724/169954095-848c83e5-613a-4297-8376-84a7a0866d75.jpg)

Reviewer: It would be good to include the image in the `labs/images` folder

Author: gr8, I've added the images into labs images path too. tnx

# Initial Committers

> _Enter the Github IDs for the set of initial committers._

Reviewer: ```suggestion
```

- https://github.com/nikoturin

> - ...

Reviewer: ```suggestion
```
